backend/schemas/source/sections/FederalAwards.schema.jsonnet

```diff
@@ -327,6 +326,7 @@ local FederalAwardEntry = Types.object {
     loan_or_loan_guarantee: Parts.LoanOrLoanGuarantee,
     program: Parts.Program,
     subrecipients: Parts.Subrecipients,
+    award_index: Types.string,
```

Comment:
This is an auto-generated key in FederalAwards.xlsx. It is required in order to (uniquely) identify each award but the 'required' flag cannot be enforced via JSON schema. The reason for this is that when the xlsx sheet is auto-filled by python, auto-generated cells (like award_index which are read-only) do not get populated, causing all award validation tests to fail. Since the "award_index" get populated correctly when manually filling the sheet I am suggesting that we validate this field exists for each award during cross-validation. FYI, a unique identifiyer will be needed for linking awards and findings. Any thought ???

Reply:
See #1514